rubin_jupyter_utils/config/rubinconfig.py

```diff
@@ -275,6 +275,18 @@ def create_derived_settings(self):
                                 ((self.external_instance_url or "") +
                                  "/portal/app/"))
         self.hub_headers = {"Authorization": "token {}".format(self.api_token)}
+        self.multus_annotation = None
+        self.multus_init_container_image = None
+        if self.enable_multus:
+            if os.getenv("MULTUS_ANNOTATION"):
+                (k, v) = os.getenv("MULTUS_ANNOTATION").split(':')
+                self.multus_annotation = {k: v}
```

Comment:
is this configuration ever used?  Do we need more annotation?

Reply:
I think there may eventually be different annotations for pods joined to different subnets.  We don't need it yet, but the summit network seems to be getting alarmingly more complex.